ruby.js

```diff
@@ -157,6 +200,14 @@ function activate(context) {
 		balancePairs(vscode.window.activeTextEditor.document);
 	}
 
+	which("rct-complete", function(err, found) {
```

Comment:
Sorry, I figure out this problem. I'm using msys2 mingw64 environment. And ruby on mingw64 doesn't allow non-absolute path in the command.